### PR TITLE
Add django admin

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -10,6 +10,5 @@ __pycache__
 *.sublime-project
 
 # project
-dashboard/settings/local.py
```
Comment (cliffxuan) [inline .dockerignore:13]: should this not be ignored?

**Comment by cliffxuan** [comment]
👍  looks good. just one little question about `.gitignore`

File: dashboard/apps/prototype/migrations/0001_initial.py [hunk 2]
```diff
@@ -1,82 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (cliffxuan) [inline dashboard/apps/prototype/migrations/0001_initial.py:1]: good move to remove this for now. might be worth adding the directory in `.gitignore` for the time being, so they don't accidentally get added in again.

File: .dockerignore (hunk 1 above)
Comment (s-block) [inline .dockerignore:13]: in docker ignore. only relevant when using docker compose

Comment (s-block) [inline .dockerignore:13]: uwsgi.ini is used on deploy atm so uses base. needs more work...

File: dashboard/apps/prototype/migrations/0001_initial.py (hunk 2 above)
Comment (s-block) [inline dashboard/apps/prototype/migrations/0001_initial.py:1]: migrations dir shouldn't get added again unless migrations are initialised. can do that when we have a more stable model.

File: .dockerignore (hunk 1 above)
Comment (s-block) [inline .dockerignore:13]: p.s. docker compose only used locally

Comment (cliffxuan) [inline .dockerignore:13]: i see. i miss read it. thought this is `.gitignore`

**Pull request merged**